vzlogger_buildroot_builder.Dockerfile

```diff
@@ -0,0 +1,236 @@
+# Dockerfile to build a buildroot-based cross-compile environment for vzlogger
+
+# the OS version used here is largely irrelevant,
+# as buildroot compiles it's own toolchain
+ARG	DEBIAN_VERSION=buster
+FROM	debian:$DEBIAN_VERSION as builder
+
+LABEL	maintainer="Thorben T. <r00t@constancy.org>"
+
+# don't inherit upstream labels
+LABEL	org.label-schema.build-date=null
+LABEL	org.label-schema.license=null
+LABEL	org.label-schema.name=null
+LABEL	org.label-schema.schema-version=null
+LABEL	org.label-schema.vendor=null
+
+# update host-OS and install required host packages
+# (we should optimize by using a base image that already contains this.)
+RUN	set -xe ; \
+	apt-get update ; \
+	apt-get -y --force-yes upgrade ; \
+	apt-get -y --force-yes install \
+		# https://buildroot.org/downloads/manual/manual.html#requirement
+		make gcc g++ \
+		patch \
+		bzip2 \
+		cpio \
+		unzip \
+		rsync \
+		file \
+		bc \
+		#libncurses-dev \
+		wget \
+		git \
+		# buildroot will build it's own cmake because it's newer than debian's
+		#cmake \
+	; \
+	apt-get purge ; \
+	rm -fr /var/lib/apt/lists
+
+WORKDIR	/buildroot
+
+RUN \
+	set -xe ; \
+	\
+	# download and unpack buildroot
+	wget --progress=dot:mega https://buildroot.org/downloads/buildroot-2021.02.1.tar.bz2 ; \
```

Comment:
I think the buildroot version should be an ARG, so that testing a other version is easy

Reply:
we could even pull buildroot for git.

i did not make this a parameter yet because i am unsure how much of the trickery below will break on updates anyway and need fixing.
i don't assume this would simply work.